data/lib/data_module.dart

```diff
@@ -0,0 +1,16 @@
+library data;
+
+export 'src/converters/from_character_to_table.dart';
+export 'src/converters/from_table_to_character.dart';
+export 'src/data_sources/local/dao/character_dao.dart';
```

Comment:
We have to export files that we are using outside `data` package ONLY. So `character_dao `, `character_table `, etc looks like can be internal. Please double check all your exports - we should export as less as possible.